src/macros.h

```diff
@@ -31,7 +31,7 @@
 #define BIGNEGFP -1.225E308
 #define flt_equals(a, b) (fabs((a) - (b)) < EPSILON)
 
-#define GETMEM(P, T, N) (P = (T*)malloc((sizeof(T) * N)))
+#define GETMEM(P, T, N) (P = (T*)malloc((sizeof(T) * (N))))
```

Comment:
Shouldn't `T` be parenthesized as well?

Reply:
I'd say we're safe with `P`, because the only operator with lower precedence than assignments is comma and that one evaluates to the right operand anyway. :grin:

Reply:
> Shouldn't `T` be parenthesized as well?

I don't think so.  Is `(double)*` valid C++?

Reply:
Ooops. :smile: